java/test/jmri/jmrix/openlcb/OlcbThrottleTest.java

```diff
@@ -409,6 +409,7 @@ public static void postClassTearDown() {
         memo = null;
         connection = null;
         nodeID = null;
+        JUnitUtil.clearShutDownManager(); // put in place because Block Manager was not terminated properly
```

Comment:
There is a specific JunitUtil method for handling the BlockManager (which I see no attempt to terminate in this test, so the comments incorrect).

Reply:
Thanks, hadn't spotted JUnitUtil.deregisterBlockManagerShutdownTask(); before.